### PR TITLE
SNAKE QOL!!!!!!!!!!!!

### PR DESCRIPTION
**Pull request by BongoBerger**
:) idk how to make options open after enabling smth or keys could be configured for snake helper ig

### CODE REVIEW

**Comment by BongoBerger** [comment]
oopsies lol

**Comment by Killermaschine88** [comment]
Needs to resolve conflicts

File: constants/settings.js [hunk 1]
```diff
@@ -211,6 +212,15 @@ class Settings {
...
     showHecatombLevel = false;
 
+    // snake helper
+    @SwitchProperty({
+        name: "Snake Minigame QOL",
+        description: "Allows you to use the WASD keys to move the snake around in the abiphone minigame.\nDefaults to §cDisabled§r.",
+        category: "Misc",
+        subcategory: "Random",
+    })
+    snakeHelper = false; // i forgot how to add options that open up after this is enabled or i would allow custom keys
```
Comment (Killermaschine88) [inline constants/settings.js:222]: change this to snakeMinigameHelper

File: features/misc/snakeHelper.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+		if (char == "a") {
+			Player.getContainer().click(50);
+		}
+		if (char == "s") {
+			Player.getContainer().click(53);
+		}
+		if (char == "d") {
+			Player.getContainer().click(52);
+		}
+	}
```
Comment (Killermaschine88) [inline features/misc/snakeHelper.js:43]: change this to use an object like

```
const keys = { w: 51, a: 50 }
```
and then just
```
const invSlot = keys[char]

if(!invSlot) return;
Player.getContainer().click(invSlot);
```
thats a way cleaner way of doing it

wont be merged before these changes are implemented

**Pull request closed without merging**